metrics/grafana/ticdc.json

```diff
@@ -106,9 +106,9 @@
       },
       {
         "datasource": "${DS_TEST-CLUSTER}",
-        "enable": false,
```

Comment:
This file changes so much, did you expand all tabs when exporting the metric file?